docs/source/developer/index.rst

```diff
@@ -49,22 +55,6 @@ precious ``.gitignore`` file.
 .. note:: Use ``openapi-generator`` version 5.2.1 or newer in order to work
    around a bug in the generator.
 
-Note that there still may be a bug in the generator, even at version 5.2.1. If
-that's the case, you can try a patched version by `Thomas Loubrieu`_ as
-follows::
-
-    cd /tmp
-    git clone https://github.com/tloubrieu-jpl/openapi-generator.git
-    cd /tmp/openapi-generator
-    ./mvnw -Dmaven.test.skip=true clean package
-
-Then instead of using ``openapi-generator`` as above, do this instead::
-
-    java -jar /tmp/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g python-legacy -i swagger.json --package-name pds.api_client --additional-properties=packageVersion=X.Y.Z
-
-again, replacing ``X.Y.Z``. Note that Thomas has reported_ this issue and made
```

Comment:
Since these lines are deleted, you can remove the `_reported` and `_pull request` references at the end of the file too 👍

Reply:
Done